src/blog/kuadrant-v1.md

```diff
@@ -0,0 +1,52 @@
+
+### Kuadrant v1!
+
+We're thrilled to announce the release of Kuadrant [v1.0.0](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.0)! This marks a major milestone in the Kuadrant project after its recent acceptance into [CNCF's sandbox](https://www.cncf.io/sandbox-projects/).
+
+### What is Kuadrant?
+
+Kuadrant is a set of Kubernetes-native controllers, services, and APIs that provide gateway policies for existing [Gateway API](https://gateway-api.sigs.k8s.io/) providers in both single and multi-cluster environments. It builds on top of Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/) and technologies such as Istio and Envoy to introduce provider-agnostic Gateway Policies for Kubernetes.
+
+### Why v1 Matters
+
+Reaching v1 signifies our project's stability, maturity, and production-readiness. With v1, we believe Kuadrant is now ready to power key workloads in Kubernetes environments.
+
+### What's new in v1?
+
+There are too many features and bug fixes to mention (although you are welcome to peruse our [release changelogs](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.0)). Here are some highlights:
+
+- **CRDs Graduation**: `AuthPolicy`, `DNSPolicy`, `RateLimitPolicy`, and `TLSPolicy` have now graduated to `v1` in their CRDs, signalling their readiness for production use.
+- **Policy Machinery Integration**: Our core components now implement [policy-machinery](https://github.com/Kuadrant/policy-machinery), providing a powerful way to visualise the "state of the world" for policies on clusters. No more guessing where policies and their behaviours originate from—policy machinery and visualisation through the console plugin make it easier than ever to reason about policy origins.
+- **Shift to the Sail Operator**: With v1, we recommend installing Istio using the new Istio [Sail Operator](https://istio.io/latest/blog/2024/introducing-sail-operator/) instead of `istioctl`.
+- **CEL Support**: [CEL](https://cel.dev/) (Common Expression Language) is now supported for [`AuthPolicy`](https://github.com/Kuadrant/authorino/blob/main/docs/features.md#common-feature-common-expression-language-cel) and [`RateLimitPolicy`](https://docs.kuadrant.io/1.0.x/kuadrant-operator/doc/reference/ratelimitpolicy/#predicate) predicates, enabling more flexible and expressive configurations.
+- **SectionName Support**: Added support for [SectionName](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.SectionName), allowing users to apply policies to specific sections of Gateways.
+- **New Console Plugin**: For [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) / [OKD](https://okd.io/) console users, a new Console Plugin is available for managing policies. We plan to make some of this functionality available through other means for Kubernetes users in the near future.
+
```

Comment:
DNS HealthChecks 
WASM Based Auth

Reply:
Although it landed earlier, would it be worth highlighting envoy gateway support?

Reply:
Sure I don't see why not